amanuensis/resources/userdatamodel/userdatamodel_project.py

```diff
@@ -135,6 +139,31 @@ def get_associated_users(current_session, emails):
         return []
     return current_session.query(AssociatedUser).filter(AssociatedUser.email.in_(emails)).all()
 
+
+def get_associated_user(current_session, email):
+    if not email:
+        return
+    return current_session.query(AssociatedUser).filter(AssociatedUser.email == email).first()
+
+
+def get_associated_users_by_id(current_session, ids):
+    if not ids:
+        return []
+    return (
+        current_session.query(AssociatedUser)
+        .filter(AssociatedUser.user_id.in_(ids))
+        .all()
+    )
+
+def get_associated_user_by_id(current_session, id):
+    if not id:
+        return
+    return (
+        current_session.query(AssociatedUser)
+        .filter(AssociatedUser.user_id == id)
+        .first()
+    )
+
```

Comment:
get_associated_user_by_id is duplicated, the same function is in the userdatamodel_associated_user file. All these 3 should be in the userdatamodel_associated_user file instead of the userdatamodel_project file.